### DISCUSSION TITLE
Is there a way to have footer columns like headers ?

### DISCUSSION THREAD

**Discussion by Pamwamba** [nick-keller/react-datasheet-grid#205] [Q&A]
Hi, I used the "addRowsComponent" prop to make my custom footer, and add a select as a footer for each columns. 

This looks perfect to me, but when I have a lot of columns, my footer goes out of the grid so I have to scroll the whole page to the right instead of scrolling the grid component (and it creates an offset between footer and column) :

![image](https://user-images.githubusercontent.com/25431260/200283951-c2b4a480-1ccf-4ff5-aa15-1b72c311061b.png)

Is there a way to make a responsive footer that matches with headers ?

**Comment by Pamwamba** [comment]
I managed to do it, I'm not proud of it. I put datasheetgrid in a div, changed overflows, adjusted some width.

I think it would be great to add possibility to have a custom footer for columns.

Let me know if you planned something like this. :)

**Comment by nick-keller** [comment]
Hey @Pamwamba ,
I might implement it in the comming months, I'll let you know

**Reply by josecao** [reply]
Hi @nick-keller, i dont find this feature in document.
